cmd/helm/helm.go

```diff
@@ -59,6 +59,34 @@ func initKubeLogs() {
 	klog.InitFlags(gofs)
 	pflag.CommandLine.AddGoFlagSet(gofs)
 	pflag.CommandLine.Set("logtostderr", "true")
+
+	// List of logging parameters used by Helm and klog, see:
+	// https://github.com/kubernetes/klog/blob/master/klog.go#L422
+	var loggingFlags = []string{
```

Comment:
can we rename this as `klogFlags`?

Is there a way we can fetch these flags from `klog` directly? I imagine as we update `klog` in future updates, more flags will be introduced.